spec/fixtures/api/default-menu/main.js

```diff
@@ -0,0 +1,20 @@
+const { app, Menu } = require('electron')
+
+let expectedMenu
+
+if (app.commandLine.hasSwitch('custom-menu')) {
+  expectedMenu = new Menu()
+  Menu.setApplicationMenu(expectedMenu)
+} else if (app.commandLine.hasSwitch('null-menu')) {
+  expectedMenu = null
+  Menu.setApplicationMenu(null)
+}
+
+app.on('ready', () => {
+  setImmediate(() => {
+    process.stdout.write(JSON.stringify(Menu.getApplicationMenu() === expectedMenu))
+    process.stdout.end()
+
+    app.quit()
+  })
+})
```

Comment:
please wrap this whole thing in a try/catch, incl. inside the `setImmediate`, so that the test will fail rather than timeout if something in here `throw`s. Makes things much easier to debug.

Reply:
done